Pod/Classes/LSClockState.m

```diff
@@ -90,10 +90,11 @@ - (void)addSampleWithOriginMicros:(SInt64)originMicros
     // Note that self.samplesQueue is DISPATCH_QUEUE_SERIAL and it makes thread safe reads/writes.
     dispatch_async(self.samplesQueue, ^{
         // Discard the oldest sample and push the new one.
-        [self.samples removeObjectAtIndex:0];
-        [self.samples
-         addObject:[[LSSyncSample alloc] initWithDelayMicros:latestDelayMicros offsetMicros:latestOffsetMicros]];
-
+        if (self.samples.count > 0) {
```

Comment:
FWIW, it's always been weird to me that this is a `> 0` sort of thing rather than `> 5` or whatever... i.e., not sure why it's an array. But the changes from today don't modify those semantics, so I'll merge.